config/database.yml

```diff
@@ -20,6 +20,16 @@ development:
     <<: *default
     database: exercism_v3_dj_development
     migrations_paths: "db/dj_migrate"
+
+development_docker:
```

Comment:
Is it worth just doing `<<: &development` in here, rather than duplicating?

Reply:
Let me try this!

Reply:
Just doing `<<: &development` doesn't work, but for the `primary` and `dj` ones it does work

Reply:
Interesting. Cool :)